DXRMapFixups/DeusEx/Classes/DXRFixupM04.uc

```diff
@@ -177,6 +178,16 @@ function PreFirstEntryMapFixes()
             ft.flagValue=True;
             ft.Event='UNATCOHatesPlayer';
 
+            foreach AllActors(class'Teleporter', tel) {
+                if (tel.url == "04_NYC_Street#FromNSFHQ") {
+                    loc = tel.Location;
+                    loc.z -= 40.0;
+                    tel.SetLocation(loc);
```

Comment:
does SetLocation actually work here? Teleporters have bStatic so I don't think they can be moved?

anyways easy enough to just make the collision height bigger without moving the teleporter, and that's probably safer too

Reply:
You're correct.